src/declarations/stencil-public-compiler.ts

```diff
@@ -1810,14 +1810,13 @@ export interface OutputTargetDist extends OutputTargetBase {
 
 export interface OutputTargetDistCollection extends OutputTargetBase {
   type: 'dist-collection';
-
+  empty: boolean;
```

Comment:
These types are a part of the public interface to Stencil (in this part the config). We can't add fields to these interfaces without making them optional, otherwise that would result in a breaking change for users. We should make this optional like the other fields we added here:
```suggestion
  empty?: boolean;
```